docs/Developers/Profile_Bears.rst

```diff
@@ -0,0 +1,74 @@
+Profile Bears
+=============
+
+This document provide the overview of coala's Profiling Interface. The
```

Comment:
provide**s** ;) --> https://www.gymglish.com/en/english-grammar/the-s-in-the-third-person-singular-form :D